lib/ledger_sync/adaptors/ledger_serializer_attribute_set.rb

```diff
@@ -4,9 +4,11 @@ module LedgerSync
   module Adaptors
     class LedgerSerializerAttributeSet
       attr_reader :attributes,
+                  :deserialize_attributes,
```

Comment:
I might call these `deserialized_attributes` and `serialized_attributes` (with a `d`), since it sounds like a method otherwise. Just my take tho

Reply:
What about `deserializable_attributes`?  I also feel like `deserialized_attributes` implies it was already done. haha

Reply:
If cool with it, approve and merge. :)